.gitignore

```diff
@@ -1,5 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+./server/client-build/index.html
```

Comment:
Probably for a separate PR, but there are a couple of files in [server/client-build](https://github.com/adobe/asset-compute-devtool/tree/NUI-502/server/client-build) that all are just build copies of [client/public](https://github.com/adobe/asset-compute-devtool/tree/NUI-502/client/public) AFAICS. IMO the entire `server/client-build` does not need to be in git.

cc @jdelbick